otel-extensions/src/test/resources/nonDefaultDataCaptureConfig.yaml

```diff
@@ -0,0 +1,23 @@
+# use snake case for newly added fields
+service_name: service
+enabled: false
+propagationFormats:
+  - B3
+reporting:
+  endpoint: http://localhost:4317
+  metric_endpoint: http://localhost:4317
+  secure: true
+  trace_reporter_type: OTLP
+  metric_reporter_type: METRIC_REPORTER_TYPE_OTLP
+  cert_file: /foo/bar/example.pem
+dataCapture:
+  bodyMaxSizeBytes: 16
+  httpHeaders:
+    request: true
+    response: false
+  httpBody:
+  allowed_content_types: ["foo", "bar"]
```

Comment:
Should we only use the ones given here or add these along with the default? I guess using only the ones given in config would give us options to remove some content-types if we do not want to capture those.

Reply:
https://github.com/hypertrace/javaagent/pull/367#discussion_r946068547

Reply:
We'd like the config to override the defaults. So if you set the config then only those in the config will be used.